pkg/transformer/openshift/openshift.go

```diff
@@ -622,23 +624,25 @@ func (o *OpenShift) Undeploy(komposeObject kobject.KomposeObject, opt kobject.Co
 			//delete pods
 			pod, err := kclient.Pods(namespace).List(options)
 			if err != nil {
-				return err
+				ErrorList = append(ErrorList, err)
 			}
 			for _, l := range pod.Items {
 				if reflect.DeepEqual(l.Labels, komposeLabel) {
 					rpPod, err := kubectl.ReaperFor(api.Kind("Pod"), kclient)
 					if err != nil {
-						return err
+						ErrorList = append(ErrorList, err)
 					}
+
 					//FIXME: gracePeriod is nil
 					err = rpPod.Stop(namespace, t.Name, TIMEOUT*time.Second, nil)
 					if err != nil {
-						return err
+						ErrorList = append(ErrorList, err)
 					}
 					log.Infof("Successfully deleted Pod: %s", t.Name)
+
 				}
 			}
 		}
 	}
-	return nil
+	return ErrorList
```

Comment:
This makes sense. Instead of Go routines (maybe implement them later if API calls start becoming lengthy), just iterating through each-step would be the best-case-scenario. I totally agree!